che-theia-init-sources.yml

```diff
@@ -22,6 +22,7 @@ sources:
     - extensions/eclipse-che-theia-remote-impl-che-server
   plugins:
     - plugins/containers-plugin
+    - plugins/ext-plugin
```

Comment:
Maybe something more descriptive? E.g. `che-api-export-plugin`

Reply:
The name of the folder can be as long as you want (can change it easily) but for the name of the plug-in it looks a bit too long/verbose for consumers.

Reply:
Agree, it's important to have the name as short as possible here. I just forgot it's supposed to be used by the external VS Code extensions.

Reply:
If you've a better short name suggestion, I can rename it

Reply:
It's good. Nevermind )